lego/apps/flatpages/tests/test_api.py

```diff
@@ -1,8 +1,28 @@
+import uuid
+
 from lego.apps.flatpages.models import Page
-from lego.apps.users.tests.utils import create_normal_user, create_super_user
+from lego.apps.users.models import AbakusGroup, User
+from lego.apps.users.tests.utils import (
+    create_normal_user,
+    create_super_user,
+    create_user_with_permissions,
+)
 from lego.utils.test_utils import BaseAPITestCase
 
 
+def get_new_unique_page():
+    return {
+        "title": str(uuid.uuid4()),
```

Comment:
tips: You can prefix these in order to make it easier to debugg when shit hits the fan. Eg `"title": f"title-{uuid.uuid4()}"`